docs/configuration.md

```diff
@@ -1,147 +1,296 @@
+  command-line `yb build targetname`, in CI configuration sections or as things
+  to build first.
+
+- `commands`: The commands to be executed; these can be anything you would run
+  on the command-line. **Note** YourBase does not expand environment variables
+  using the normal `${VALUE}` syntax, though there are ways to do this.
+  We do not run `sh` or `bash` as this is not portable; if you need to do that,
+  create a shell script and run it as a command explicitly.
+
+- `dependencies`: See the [Dependencies section](#dependencies).
+
+- `environment`: A list of `KEY=VALUE` items that are used as environment
+  variables.
+
+  ```yaml
+  build_targets:
+    - name: default
+      environment:
+        - ENVIRONMENT=development
+        - DATABASE_URL=db://localhost:1234/foo
+  ```
+
+- `root`: The name of a path where it should run commands from, relative to the
+  root of the project.
+
+- `build_after`: A list of build targets whose commands will be executed before
+  this target.
+
+  ```yaml
+  build_targets:
+    - name: test
+      commands:
+        - python tests.py
+    - name: release
+      build_after:
+        - test
+      commands:
+        - python kraken.py
+  ```
+
+- `container`: If present, all the commands in the `commands` directive will
+  be executed inside a Docker container. See the [Containers section](#containers)
+  for details on the attributes.
+
+  ```yaml
+  build_targets:
+    - name: container_build
+      container:
+        image: golang:1.12
+      commands:
+        - go get
+        - go build
+        - go test ./...
+  ```
+
+  Using a container is the preferred way of installing software that has complex
+  dependencies. For example, if you want to use an `apt-get` installable package
+  in your build:
+
+  ```yaml
+  build_targets:
+    - name: container_build
+      container:
+        image: yourbase/yb_ubuntu:18.04
+      environment:
+        - DEBIAN_FRONTEND=noninteractive
+      commands:
+        - apt-get update
+        - apt-get install -y --no-install-recommends cowsay
+        - cowsay moo
+  ```
+
+- `tags`: Tags are used to select which build server can handle each build in CI.
+
+  - `os`: Specifies which operating system this build target is applicable to.
+    Current valid options: `linux` (default) and `osx`.
+
+    ```yaml
+    - name: android
+      tags:
+        os: linux
+      commands:
+        - apt-get update
+        - apt-get install -y software-properties-common
+        - add-apt-repository -y ppa:ubuntu-toolchain-r/test
+        - apt-get update
+        - apt-get install -y lib32stdc++6
+        - sh build.sh
+    - name: ios
+      tags:
+        os: darwin
+      commands:
+          - brew update
+          - brew install --HEAD usbmuxd
+          - brew link usbmuxd
+          - brew install --HEAD libimobiledevice
+          - brew install ideviceinstaller
+          - brew install ios-deploy
+          - sh build.sh
```

Comment:
```suggestion
        - brew update
        - brew install --HEAD usbmuxd
        - brew link usbmuxd
        - brew install --HEAD libimobiledevice
        - brew install ideviceinstaller
        - brew install ios-deploy
        - sh build.sh
```

Reply:
Thanks! Sorry for the auto-merge. I applied your fix in 753e9b9c2e01ab7ed11683cbcc9cef51377030f3.